building/build-acis/common/container-build-helpers.sh

```diff
@@ -53,6 +57,7 @@ function allocate_tempdir() {
 		echo "Could not create temporary directory." 1>&2
 		exit 1
 	fi
+	B="${TMPBUILDDIR}"
```

Comment:
Why reassign TMPBUILDDIR? Presumably B would always be the same as TMPBUILDDIR, since the if statement exits. Also, I have reservations about the variable name being non-descriptive and hard to grep, but I can see the case for it being short here, so I won't let this block it.

Reply:
Line 54 is the only place that TMPBUILDDIR is assigned.

B is a shorthand. You can grep for it as either `${B}` or `$B[^A-Z]`. I also have similar reservations, but it seemed worthwhile.